community/hangouts/index.md

```diff
@@ -10,19 +10,19 @@ permalink: /community/hangouts/index.html
     <section class="hangouts-social container">   
         <ul class="row center-xs">
             <li>
-                <a href="https://twitter.com/monero" target="_blank" rel="noreferrer noopener"><div class="social-icon twitter"></div></a>
+                <a id="ext-noicon" href="https://twitter.com/monero" target="_blank" rel="noreferrer noopener"><div class="social-icon twitter"></div></a>
```

Comment:
Having multiple elements with the same `id` is invalid HTML.

```suggestion
                <a class="ext-noicon" href="https://twitter.com/monero" target="_blank" rel="noreferrer noopener"><div class="social-icon twitter"></div></a>
```

Reply:
Fixed. Thanks.